providers/in-process-flagd/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: mockgen
+mockgen:
+	mockgen -source=pkg/service/iservice.go -destination=internal/mock/service_mock.go -package=mock
```

Comment:
🤔 not sure this was added by mistake, but source doesn't exisit

Reply:
yes, good catch, that was indeed added by accident - removed it